src/org/labkey/test/selenium/LazyWebElement.java

```diff
@@ -39,6 +41,19 @@ public LazyWebElement(@NotNull Locator locator, @NotNull SearchContext searchCon
         _searchContext = searchContext;
     }
 
+    @Override
+    public boolean isDisplayed()
+    {
+        try
+        {
+            return super.isDisplayed();
+        }
+        catch (StaleElementReferenceException | NoSuchElementException ex)
+        {
+            return false;
+        }
+    }
+
```

Comment:
There are a handful of places that use 'isDisplayed' to trigger a staleness check (which this breaks). Some are in [testAutomation](https://github.com/LabKey/testAutomation/pull/478), [adjudication](https://github.com/LabKey/adjudication/pull/44), and [sample manager](https://github.com/LabKey/sampleManagement/pull/371).
I checked the CDS changes that go along with this and you could tweak the existing assertions and not need the `isDisplayed` change. They would look something like:
```
checker().wrapAssertion(() -> assertElementPresent(Locator.tagWithClass("div", "detail-gray-text").containing("HIV")));
```

Reply:
I was unaware of that.
I've reverted this change.